tests/test_synchronize_schemas.py

```diff
@@ -0,0 +1,131 @@
+import synapseclient as sc
+import uuid
+from synapsebridgehelpers import synchronize_schemas
+from copy import deepcopy
+
+syn = sc.login()
+
+TESTING_TABLE = "syn18503475"
+TESTING_TABLE_2 = "syn18503678"
+TEMP_PROJECT = "syn11657334"
+TESTING_TABLE_ORIGINAL = syn.tableQuery(
+        "select * from {}".format(TESTING_TABLE)).asDataFrame()
+TESTING_TABLE_2_ORIGINAL = syn.tableQuery(
+        "select * from {}".format(TESTING_TABLE_2)).asDataFrame()
+TESTING_TABLE_ORIGINAL_COLS = list(syn.getTableColumns(TESTING_TABLE))
+TESTING_TABLE_2_ORIGINAL_COLS = list(syn.getTableColumns(TESTING_TABLE_2))
+
+def create_testing_table(syn, cols, values):
+     testing_schema = sc.Schema(name = str(uuid.uuid4()),
+                                columns = cols,
+                                parent = TEMP_PROJECT)
+     testing_table = sc.Table(testing_schema, values)
+     testing_table = syn.store(testing_table)
+     return testing_table
+
+def delete_testing_tables(syn, testing_tables):
+   for t in testing_tables:
+       syn.delete(t)
+
+def test_add_column():
+    source_table = create_testing_table(
+            syn, TESTING_TABLE_ORIGINAL_COLS, TESTING_TABLE_ORIGINAL)
+    target_table = create_testing_table(
+            syn, TESTING_TABLE_ORIGINAL_COLS, TESTING_TABLE_ORIGINAL)
+    try:
+        source_table = syn.get(source_table.tableId)
+        target_table = syn.get(target_table.tableId)
+        new_column = sc.Column(columnType="STRING", maximumSize=5, name="new_col")
+        source_table.addColumn(new_column)
+        source_table = syn.store(source_table)
+        target_table = synchronize_schemas(
+                syn,
+                schema_comparison = {"added": ["new_col"]},
+                source = source_table["id"],
+                target = target_table["id"])
+        source_cols = [c["name"] for c in syn.getTableColumns(source_table["id"])]
+        target_cols = [c["name"] for c in syn.getTableColumns(target_table["id"])]
+        assert all([c in source_cols for c in target_cols])
+    finally:
+        delete_testing_tables(syn, [source_table, target_table])
```

Comment:
use test set-up and teardown to create temporary data structures - see e.g. python client test suite.

Reply:
note to self: TODO